gef.py

```diff
@@ -3257,6 +3304,213 @@ def del_setting(self, name):
 #     def do_invoke(self, argv):
 #         return
 
+@register_command
+class PieCommand(GenericCommand):
+    """PIE breakpoint support"""
+
+    _cmdline_ = "pie"
+    _syntax_  = "{:s} (breakpoint|info|delete|run|attach|remote)".format(_cmdline_)
+
+    def __init__(self):
+        super(PieCommand, self).__init__(prefix=True)
+        return
+
+    def do_invoke(self, argv):
+        if len(argv)==0:
+            self.usage()
+            return
```

Comment:
This doesn't really matter, but this should be untabbed if we want to be consistent with hugsy's style where functions end with `return`